### ISSUE TITLE
Trouble with 'styled-jsx/webpack' Loader and Storybook

### ISSUE THREAD

**Issue by ryanfitzer** [zeit/styled-jsx#498]
#### Do you want to request a *feature* or report a *bug*?

Bug

#### What is the current behavior?

Using the new feature that enables importing regular css files causes Storybook to error:

```
styled-jsx/css: if you are getting this error it means that your `css` tagged template literals were not transpiled.
```

#### If the current behavior is a bug, please provide the steps to reproduce and possibly a minimal demo or testcase in the form of a Next.js app, CodeSandbox URL or similar

I've created an example project that shows the issue: https://github.com/ryanfitzer/styled-jsx-storybook-test

#### What is the expected behavior?

Storybook will render the components without a `styled-jsx` error.

#### Environment (include versions)

* OS: macOS 10.12.6 (16G1510)
* Browser: Google Chrome Version 69.0.3497.100 (Official Build) (64-bit)
* styled-jsx (version): v3.0.3-canary.0 (ships with next v7.0.0)

#### Did this work in previous versions?

Feature wasn't available in previous versions.

**Comment by giuseppeg** [comment]
in your Storybook webpack config you need to invert the plugins order i.e. run `styled-jsx/webpack` before `babel-loader` and then also add `styled-jsx/babel` to the babel loader plugins I guess. Note that in order to transpile the `.css` file the `babel-loader` `test` might need to match `.css` too eg: `test: /\.(jsx?|css)$/`

**Comment by aoc** [comment]
@ryanfitzer 

I may be wrong, but I think the problem is that storybook uses babel 6 while styled-jsx uses 7.
You can launch storybook but it's not processing the <style jsx> tags and it's not generating the unique classes names.

I switched to storybook 4 alpha to fix that:

`package.json`
```json
"devDependencies": {
    "@storybook/react": "^4.0.0-alpha.23",
    "@storybook/addon-actions": "^4.0.0-alpha.23",
    "@storybook/addon-links": "^4.0.0-alpha.23",
    "@storybook/addons": "^4.0.0-alpha.23"
  }
```

and added `.storybook/.babelrc`
```
{
  "presets": ["next/babel"]
}
```

hope that works for you

**Comment by giuseppeg** [comment]
styled-jsx itself is still written for babel 6 (although it is mostly compatible with babel 7), but next yep it uses babel 7

**Comment by giuseppeg** [comment]
@ryanfitzer if you are using `next/babel` then you don't need to add `styled-jsx/babel` since the preset includes styled-jsx already

**Comment by ryanfitzer** [comment]
@giuseppeg Yep, you're right. Just tested it in the example repo and everything worked correctly.

So the 2 issues that fixed it:

1. Making sure the the `styled-jsx/webpack` loader runs before `'babel-loader'` (positioned last in the `rules` array)
2. Adding  the `css` extension the `babel-loader`'s `test` property: `test: /\.(jsx?|css)$,`.

Thanks, again! I pushed changes to the [example repo](https://github.com/ryanfitzer/styled-jsx-storybook-test/blob/master/storybook/webpack.config.js).

**Comment by nhayhoc** [comment]
@ryanfitzer  you save my life !

**Comment by Brewd3v** [comment]
@ryanfitzer  Thanks!
